src/app/(Authentication)/SignUp/Email/styles.ts

```diff
@@ -71,14 +71,15 @@ export default StyleSheet.create({
     padding: 15,
     flexDirection: 'row',
   },
+  errorMessageContainer: {
+    width: '75%',
+    marginTop: 10,
+    marginBottom: 65,
+  },
```

Comment:
What is we made an `AuthErrorContainer` component. It'd take the error text as a prop and would allow us to easily standardize styling throughout the flow.